jans-config-api/plugins/admin-ui-plugin/src/main/java/io/jans/ca/plugin/adminui/service/webhook/WebhookService.java

```diff
@@ -326,6 +338,14 @@ public void validateWebhookEntry(WebhookEntry webhookEntry) throws ApplicationEx
                 throw new ApplicationException(Response.Status.BAD_REQUEST.getStatusCode(), ErrorResponse.WEBHOOK_CONTENT_TYPE_REQUIRED.getDescription());
             }
         }
+        if (Lists.newArrayList("http://", "file://", "localhost", "127.0.", "192.168.", "172.").contains(webhookEntry.getUrl())) {
```

Comment:
Should this be configurable? 
Also lets say webhook url is `http://192.168.0.1`. Code above is not going to catch it because it's not in the list. Add simple unit test to confirm it works.